google-cloud-language/lib/google/cloud/language/annotation.rb

```diff
@@ -241,8 +241,8 @@ def self.from_grpc grpc
         #
         # @attr_reader [String] text The content of the output text.
         # @attr_reader [Integer] offset The API calculates the beginning offset
-        #   of the content in the original document according to the `encoding`
-        #   specified in the API request.
+        #   of the content in the original document according to the default
+        #   encoding of the system. Ruby defaults to UTF-8.
```

Comment:
I think all `offset` attrs/methods need a full discussion of encoding. (Here's my logic: the encoding only affects one thing, the offset. So you will only care about changing encoding when you have issues with offset. And when you have issues with offset, you will look at the method docs for offset.)